modules/shared.py

```diff
@@ -106,7 +106,7 @@
 
 # bitsandbytes 4-bit
 group = parser.add_argument_group('bitsandbytes 4-bit')
-group.add_argument('--load-in-4bit', action='store_true', help='Load the model with 4-bit precision (using bitsandbytes).')
+group.add_argument('--load-in-4bit', action='store_true', help='Load the model with 4-bit precision (using bitsandbytes or ipex-llm).')
```

Comment:
"using bitsandbytes or ipex-llm" is a bit confusing. Should we tell user in which case to use bitesandbytes and in which case ipex-llm?